### ISSUE TITLE
new introduction video!

### ISSUE THREAD

**Issue by jywarren** [publiclab/mapknitter#94]
Based on the current one at http://mapknitter.org

**Comment by steviepubliclab** [comment]
Oh please!! I use this in trainings all the time.

**Comment by jywarren** [comment]
don't worry, this has been planned all along; we were just waiting until the site was launch-ready so the video doesn't immediately become out of date again.

**Comment by jywarren** [comment]
<iframe width="640" height="480" src="https://www.youtube.com/embed/0h6MBH6f2js?rel=0&amp;showinfo=0" frameborder="0" allowfullscreen></iframe>

https://www.youtube.com/watch?v=0h6MBH6f2js check it out -- the audio cuts in a little jarringly, but the rest seems pretty good. Stevie?

**Comment by jywarren** [comment]
Hi Stevie - would you be able to edit the video transcript so that we can offer subtitles, and ask organizers to translate the subtitles as well? http://pad.publiclab.org/p/mapknitter-subtitles

I think it'd make a big difference for accessibility.

**Comment by steviepubliclab** [comment]
Yes! Love it, but why doesn't your image zoom with you at the end?

Do you want it edited right on the pad? (how did you grab the text that's
there?)

**Comment by jywarren** [comment]
yeah my mac kinda froze a bit at the end, it's not really up to making
screencasts. The pad has the "auto transcription" which, if we leave the
timings and formatting the same, we should be able to upload back up and
it'll overwrite the poor auto-transcribed version. That's also the format
we can use to upload the spanish language version.

**Comment by jywarren** [comment]
I could do another recording but this one was pretty good, and I have to
make several more videos, so time is a bit short. I think it'd be great to
do one for exporting, one for "advanced mapmaking" etc. Maybe we can
inspire some community members to make these.
